src/main/java/carpetextra/dispenser/behaviors/ToggleBlockDispenserBehavior.java

```diff
@@ -52,7 +52,7 @@ protected ItemStack dispenseSilently(BlockPointer pointer, ItemStack stack) {
             BlockHitResult hitResult = new BlockHitResult(Vec3d.of(frontBlockPos), dispenserFacing.getOpposite(), frontBlockPos, false);
 
             // try to use block
-            if(frontBlockState.onUse(world, null, Hand.MAIN_HAND, hitResult).isAccepted()) {
+            if(frontBlockState.onUseWithItem(stack, world, null, Hand.MAIN_HAND, hitResult).isAccepted()) {
```

Comment:
broken?: this does nothing, the stick does not shoot out, but it doesn't toggle the sate. Not sure -what- this is supposed to do, as using this with buttons/levers would lock the dispenser? Maybe it was also broken in prior versions?

Reply:
https://github.com/voidpointer0x00/carpet-extra/pull/2
I also added the new wood type buttons